test/TorchSharpTest/NN.cs

```diff
@@ -444,6 +444,58 @@ public void EvalLossSequence()
 
             var result = output.ToSingle();
         }
+
```

Comment:
Given my comments on loading and saving earlier, please add a unit test to TestLoadSave that saves and restores the state. Something like 'TestSaveLoadConv2D' except using a Sequential instead of Conv2D as the test.